jobsdb/jobsdb.go

```diff
@@ -676,6 +681,7 @@ func loadConfig() {
 	*/
 	config.RegisterFloat64ConfigVariable(0.8, &jobDoneMigrateThres, true, "JobsDB.jobDoneMigrateThres")
 	config.RegisterFloat64ConfigVariable(5, &jobStatusMigrateThres, true, "JobsDB.jobStatusMigrateThres")
+	config.RegisterFloat64ConfigVariable(0.5, &jobMinRowsMigrateThres, true, "JobsDB.jobMinRowsMigrateThres")
```

Comment:
Please excuse me for misleading you with my initial comment with respect to this value... assuming that we consider as small tables, tables that hold less than `5%` of the max DS size, then this value should be `0.05`. Hope I didn't mess it up again :)